frontend/src/app/main/ui/workspace/shapes/frame.cljs

```diff
@@ -51,32 +51,45 @@
         ;; DEBUG
         :style {:filter (when (debug? :thumbnails) "sepia(1)")}}])))
 
-;; This custom deferred don't defer rendering when ghost rendering is
+(mf/defc frame-placeholder
+  {::mf/wrap-props false}
+  [props]
+  (let [{:keys [x y width height fill-color] :as shape} (obj/get props "shape")]
+    (if (some? (:thumbnail shape))
+      [:& thumbnail {:shape shape}]
+      [:rect {:x x :y y :width width :height height :style {:fill (or fill-color "white")}}])))
```

Comment:
I think we can use the new color constants here

Reply:
You're right. :+1: